src/mkdocs_include_markdown_plugin/process.py

```diff
@@ -117,7 +117,17 @@
 MARKDOWN_HTML_IMAGE_REGEX = re.compile(
     r'''
         <img
-        [^>]*
+        (?:\s+             # More than one whitespace
+            (?!src=)       # Not src=
+            [\w-]+         # attribute name
+            \s*=\s*        # arbitrary whitespace
+            (?:
+                "[^"]*"    # Quoted value
+                |
+                '[^']*'    # Quoted value
+            )
```

Comment:
Sometimes there are containers with attributes in the form `<img attr src="">`. Could you make this part after arbitrary whitespace optional?

Reply:
Thank you! It is fixed now.